srcpkgs/upower/template

```diff
@@ -29,6 +29,8 @@ build_options_default="gir"
 
 if [ -z "$CROSS_BUILD" ]; then
 	build_options_default+=" gtk_doc"
+else
+	hostmakedepends+=" polkit"
```

Comment:
this can be unconditional, on not-cross it's being installed for the host as a dependency of `polkit-devel`